src/test/java/teammates/test/cases/logic/EmailGeneratorTest.java

```diff
@@ -39,13 +40,15 @@
     private static final InstructorsLogic instructorsLogic = InstructorsLogic.inst();
     private static final StudentsLogic studentsLogic = StudentsLogic.inst();
 
-    /** indicates if the test-run is to use GodMode. */
-    private static boolean isGodModeEnabled;
-
-    @BeforeClass
-    public void classSetup() {
-        if (isGodModeEnabled) {
-            System.setProperty("godmode", "true");
+    /**
+     * Reminder to disable GodMode and re-run the test.
+     */
+    @AfterSuite
```

Comment:
It gets printed in an inconspicuous position though; the user needs to scroll up to see it.